### DISCUSSION TITLE
Auto change alignment position on mouse hover

### DISCUSSION THREAD

**Discussion by damywise** [mulaRahul/keyviz#7] [General]
**Is your feature request related to a problem? Please describe.**
Maybe when presenting, the thing I want to show is in the bottom right and the keyviz is also on the bottom right. I want an option in the settings page/window for it to automatically move to the other side (in this case, the bottom left side) when the keyviz is hovered with a mouse.

**Describe the solution you'd like**
Add on option like "Change alignment On Mouse Hover: [Left Side]" where the [Left Side] could be the opposite direction of the currently set alignment.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by mulaRahul** [comment]
Mouse events will be added in the future. I'll make sure to add it to future plans.
